cdap-api/src/main/java/co/cask/cdap/api/metrics/RuntimeMetrics.java

```diff
@@ -23,9 +23,9 @@
 
 /**
  * Provides access to metrics about running processors.
- *
- * TODO: Support custom metrics https://issues.cask.co/browse/CDAP-765
+ * deprecated, Usages in MetricsClient and RuntimeStats will be replaced alternative implementation.
```

Comment:
I think it should be similar to deprecation description from other class:
`This is deprecated, should use {@link co.cask.cdap.test.MetricsManager} instead`

Is that description correct?